api/server.js

```diff
@@ -1,3 +1,83 @@
 // BUILD YOUR SERVER HERE
+const express = require("express");
+const { find, findById, insert, update, remove } = require("./users/model.js");
 
-module.exports = {}; // EXPORT YOUR SERVER instead of {}
+const server = express();
+
+server.get("/api/users", async (req, res) => {
+  const users = await find();
+
+  if (users) {
+    return res.status(200).json({ users });
+  } else
+    return res
+      .status(500)
```

Comment:
404 deoarece nu ai gasit nimic

Reply:
in conditie scrie ca trebuie de returnat 500
![image](https://user-images.githubusercontent.com/51832550/107755813-fbc5d600-6d2b-11eb-92db-0abb7cf63d84.png)

Reply:
da, daca scoaterea de date din baza de date nu a mers bine, uite la comentariul de mai jos, am dat un exemplu